apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -0,0 +1,1004 @@
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Dispatch } from "redux";
+import { Dropdown, DropdownItemProps, DropdownProps, Form, Grid, Icon, Modal } from "semantic-ui-react";
+import { v4 as uuidv4 } from "uuid";
+import { getUserStores } from "../../../../extensions/components/users/api";
+import { UsersConstants } from "../../../../extensions/components/users/constants";
+import { userConfig } from "../../../../extensions/configs";
+import { getAllExternalClaims, getDialects, getSCIMResourceTypes } from "../../../claims/api";
+import { UserStoreDetails, UserStoreProperty, getCertificateIllustrations } from "../../../core";
+import { PRIMARY_USERSTORE } from "../../../userstores/constants";
+import { addBulkUsers } from "../../api";
+import {
+    BlockedBulkUserImportAttributes,
+    BulkUserImportStatus,
+    RequiredBulkUserImportAttributes,
+    SpecialMultiValuedComplexAttributes,
+    UserManagementConstants
+} from "../../constants";
+import {
+    BulkResponseSummary,
+    BulkUserImportOperationResponse,
+    BulkUserImportOperationStatus,
+    SCIMBulkEndpointInterface,
+    SCIMBulkOperation,
+    SCIMBulkResponseOperation
+} from "../../models";
+import { BulkImportResponseList } from "../bulk-import-response-list";
+
+/**
+ * Prototypes for the BulkImportUserWizardComponent.
+ */
+interface BulkImportUserInterface extends TestableComponentInterface {
+    closeWizard: () => void;
+    userstore: string;
+    ["data-componentid"]?: string;
+}
+interface CSVAttributeMapping {
+    attributeName: string;
+    mappedLocalClaimURI: string;
+    mappedSCIMAttributeURI: string;
+    mappedSCIMClaimDialectURI: string;
+    claimURI?: string;
+}
+
+interface MultiValuedComplexAttribute {
+    [key: string] : string | boolean; 
+}
+
+type ValidationError = {
+    descriptionKey: string;
+    messageKey: string;
+    descriptionValues?: { [key: string]: string };
+};
+
+interface Validation {
+    check: () => boolean;
+    error: ValidationError;
+}
+
+const ASK_PASSWORD_ATTRIBUTE: string = "identity/askPassword";
+const CSV_FILE_PROCESSING_STRATEGY: CSVFileStrategy = new CSVFileStrategy(
+    undefined,  // Mimetype.
+    userConfig.bulkUserImportLimit.fileSize * CSVFileStrategy.KILOBYTE,  // File Size.
+    userConfig.bulkUserImportLimit.userCount  // Row Count.
+);
+const DATA_VALIDATION_ERROR: string = "Data validation error";
+const ADDRESS_HOME_ATTRIBUTE: string = "addresses#home";
+const ADDRESS_ATTRIBUTE: string = "addresses";
+const HOME_ATTRIBUTE: string = "home";
+
+/**
+ *  BulkImportUserWizard component.
+ *
+ * @param props - Props injected to the component.
+ * @returns BulkImportUser
+ */
+export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> = (
+    props: BulkImportUserInterface
+): ReactElement => {
+    const { closeWizard, userstore, ["data-componentid"]: componentId } = props;
```

Comment:
```suggestion
    const {
        closeWizard,
        userstore,
        ["data-componentid"]: componentId
    } = props;
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/6597.